enterprise/management/commands/nudge_dormant_enrolled_learners.py

```diff
@@ -144,9 +172,13 @@ def get_query_results_from_snowflake(self):
         cs = ctx.cursor()
```

Comment:
We can use [DictCursor](https://docs.snowflake.com/en/user-guide/python-connector-example.html#using-dictcursor-to-fetch-values-by-column-name). For each row this will return a `dict` and you can values by specifying the column names instead of using index numbers.

Reply:
I think we don't need batch as we will not be using `fetchall`. 
```
cur = con.cursor(DictCursor)
try:
    cur.execute(QUERY)
    for row in cur:
        yield row
finally:
    cur.close()
```

Reply:
@HammadAhmadWaqas DictCursor ... "This is a Cursor class that returns rows as dictionaries and stores the result set in the client."
http://mysql-python.sourceforge.net/MySQLdb-1.2.2/public/MySQLdb.cursors.DictCursor-class.html

Reply:
@irfanuddinahmad there is [fetchManyDict](http://mysql-python.sourceforge.net/MySQLdb-1.2.2/public/MySQLdb.cursors.CursorDictRowsMixIn-class.html#fetchmanyDict) property. it says we can use fetchMany as well as fetchManyDict is deprecating.